documentation/src/docs/transactions.md

```diff
@@ -44,9 +44,9 @@ below:
 **ConsoleWriter Help text**
 ```
 Enter one of the following commands at the command line prompt:
- 
```

Comment:
There were a bunch of these lines with trailing whitespace that my text editor automatically trimmed. I can revert them if y'all want a better diff.